app/pulp/app/urls.py

```diff
@@ -21,13 +11,18 @@
 router = routers.DefaultRouter(
     schema_title='Pulp API',
     schema_url='/api/v3'
-)
+) #: The Pulp Platform v3 API router, which can be used to manually register ViewSets with the API.
```

Comment:
This is how you comment attributes, the #: is necessary. Without it, the router doesn't show up in the docs.